lib/SPIRV/OCLUtil.h

```diff
@@ -543,6 +544,79 @@ getOrCreateSwitchFunc(StringRef MapName, Value *V,
   return addCallInst(M, MapName, Ty, V, nullptr, InsertPoint);
 }
 
+/// Performs conversion from OpenCL memory_scope into SPIR-V Scope.
+///
+/// Supports both constant and non-constant values. To handle the latter case,
+/// function with swtich..case statement will be insterted into module which
```

Comment:
```suggestion
/// function with switch..case statement will be inserted into module which
```

Same typos occur further down.

Reply:
Fixed, thanks!